Modules/_struct.c

```diff
@@ -2131,6 +2131,10 @@ pack(PyObject *self, PyObject **args, Py_ssize_t nargs, PyObject *kwnames)
     PyObject *s_object = NULL;
     PyObject *format, *result;
 
+    if (!_PyArg_NoStackKeywords("pack", kwnames)) {
```

Comment:
It is needed if we want the `pack` (and `pack_into`) function to have the same behavior as other function, ie checking for (unsupported) keyword arguments is the first check we perform. If we remove it, the tests `test_varargs8_kw` and `test_varargs9_kw` fail with error "missing format argument" as the check for keyword happened afterward (in the "s_"-prefixed functions).
I'm happy to get rid of the check and remove/update the test if you think it is better.

Thanks for your feedback!